ctakesclient/client.py

```diff
@@ -20,26 +20,27 @@ def get_url_ctakes_rest() -> str:
 
     :return: URL_CTAKES_REST env variable or default using localhost
     """
-    return os.environ.get(
-        'URL_CTAKES_REST',
-        'http://localhost:8080/ctakes-web-rest/service/analyze')
+    url = os.environ.get('URL_CTAKES_REST')
```

Comment:
This change was made to allow for the common pattern of setting `URL_CTAKES_REST=` in an environment to "clear" it. (i.e. we don't want to interpret an empty string as an actual url)